dev-qt/qtwebengine/metadata.xml

```diff
@@ -14,6 +14,7 @@
 		<flag name="designer">Install the QWebEngineView plugin used to add widgets in <pkg>dev-qt/designer</pkg> forms that display web pages.</flag>
 		<flag name="geolocation">Enable physical position determination via <pkg>dev-qt/qtpositioning</pkg></flag>
 		<flag name="jumbo-build">Combine source files to speed up build process.</flag>
+		<flag name="pipewire">Enable PipeWire support for WebRTC.</flag>
```

Comment:
In 5.15, this is 'screencast' IUSE flag instead which is probably more accurate.

Reply:
Done.

Reply:
Done more better now...